frontend/src/core/network/connection.ts

```diff
@@ -1,14 +1,15 @@
 /* Copyright 2024 Marimo. All rights reserved. */
 import { atom } from "jotai";
 import { waitFor } from "../state/jotai";
+import { isStaticNotebook } from "../static/static-state";
 import { type ConnectionStatus, WebSocketState } from "../websocket/types";
 
 /**
  * Atom for storing the connection status.
- * Initialized to CONNECTING.
+ * Initialized to CONNECTING for normal mode, OPEN for static mode.
  */
 export const connectionAtom = atom<ConnectionStatus>({
-  state: WebSocketState.CONNECTING,
+  state: isStaticNotebook() ? WebSocketState.OPEN : WebSocketState.CONNECTING,
```

Comment:
This feels suspicious because it only appears to be needed when serving HTML with local assets; might not be the right fix.

Reply:
Will test locally, but looks fine